### PR TITLE
JAMES-2981 update changelog

### PR DESCRIPTION
**Pull request by remk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Multiple changes have been made to enhance ElasticSearch performance:
   - Use of routing keys to collocate documents per mailbox
   - Read related [upgrade instructions](upgrade-instructions.md)
+  - By default the cassandra keyspace creation by James is now disabled by default. This allow to have credentials limited to a keyspace. It can be enabled by setting cassandra.keyspace.create=true in the cassandra.properties file.
```
Comment (rouazana) [inline CHANGELOG.md]: can you add a note in upgrade notes please?

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,7 +18,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
   - JDBCRecipientRewriteTable implementation. Please use JPARecipientRewriteTable instead.
   - JamesUsersJdbcRepository, DefaultUsersJdbcRepository. Please use JpaUsersRepository instead.
   - MailboxQuotaFixed matcher. Please use IsOverQuota instead.
-- UsersFileRepository, which was marked as deprecated for years
+  - UsersFileRepository, which was marked as deprecated for years
```
Comment (rouazana) [inline CHANGELOG.md]: no this was on purpose, it's not relate to JAMES-2703

Comment (remk) [reply CHANGELOG.md]: ah ok, i tought it was an alignment problem.

**Comment by chibenwa** [comment]
See https://github.com/linagora/james-project/pull/2819

**Pull request closed without merging**